src/Service.Tests/SqlTests/SqlTestHelper.cs

```diff
@@ -150,6 +150,21 @@ public static void TestForErrorInGraphQLResponse(string response, string message
             }
             else
             {
+                // Json Property in error which the holds the actual exception properties.
```

Comment:
This comment needs to be revised.

Reply:
Like do you mean is it indicating something wrong?

Reply:
just needed rewording. not indicating anything wrong.